#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_MakefileFrag_patch.dpatch by  Andre Felipe Machado <af-machado@uol.com.br>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: To avoid complete substitution of original file and improve flexibility.

@DPATCH@
diff -urNad php-java-bridge-3.2.1~/Makefile.frag php-java-bridge-3.2.1/Makefile.frag
--- php-java-bridge-3.2.1~/Makefile.frag	2006-12-07 08:49:16.000000000 -0200
+++ php-java-bridge-3.2.1/Makefile.frag	2006-12-07 11:09:41.000000000 -0200
@@ -1,4 +1,8 @@
 # -*- mode: Makefile; -*-
+tmp_prefix:=$(prefix)
+prefix = ${DESTDIR}$(tmp_prefix)
+TMP_EXTENSION_DIR:=$(EXTENSION_DIR)
+EXTENSION_DIR=${DESTDIR}$(TMP_EXTENSION_DIR)
 
 $(phplibdir)/stamp:
 	cd $(srcdir)/server; $(MAKE) CFLAGS="$(CFLAGS_CLEAN)" GCJFLAGS="$(GCJFLAGS) `echo $(CFLAGS_CLEAN)|sed 's/-D[^ ]*//g'`" install
